### ISSUE TITLE
conflicts with elastic skin button message part, seems jquery issue pending

### ISSUE THREAD

**Issue by mckaygerhard** [conversejs/roundcube-plugin#14]
i made that the plugin can work, the plugin loads.. but it has some issues with the butons of the messsages: it disapears

i upload a no jquery version build by me (i really dont know how to raised that .. i just do so fast and uglyway.. no interest right now) this is the first error, i guess i some kind of confflict with the version of the roundcube.. .. 

```
Uncaught Error: See almond README: incorrect module build, no module name
    define jQuery
    <anonymous> http://localhost/skins/elastic/deps/bootstrap.bundle.min.js?s=1655027994:6
    <anonymous> https://localhost/skins/elastic/deps/bootstrap.bundle.min.js?s=1655027994:6
converse.nojquery.min.js:212:2755
```

![imagen](https://user-images.githubusercontent.com/1571724/179512568-748cf717-0912-458e-8535-2f4dcd1fe6c0.png)

for the interest:

![imagen](https://user-images.githubusercontent.com/1571724/179511213-d65ba7cc-a8a4-45d9-b688-981ac3e30f9b.png)

**Comment by drlight17** [comment]
old version issue. closed.

**Comment by mckaygerhard** [comment]
ok solved with commit 5b14f73919439533985c6f7a8c15c8d365b3c5c0

**Comment by drlight17** [comment]
> ok solved with commit [5b14f73](https://github.com/conversejs/roundcube-plugin/commit/5b14f73919439533985c6f7a8c15c8d365b3c5c0)

also check this one https://github.com/conversejs/roundcube-plugin/pull/16/commits/4224bc51ec10a8652e15173fc9a03c82ef32857f
